tasks/giovanni-adapter/app/cli.ts

```diff
@@ -99,6 +99,9 @@ export default async function main(args: string[]): Promise<void> {
     'title': 'giovanni stac item',
   });
   const stacItemFilename = path.join(options.harmonyMetadataDir, stacItemRelativeFilename);
+  const time_start = operation.temporal.start;
+  const time_end = operation.temporal.end;
+  const properties = { start_datetime: time_start, end_datetime: time_end, datetime: null };
```

Comment:
Why do we need to set `datetime` to null?

Reply:
I was just trying to follow the results from other services like `netcdf-to-zarr` and honestly I don't why it was null either, but I just left it there for consistency and in case some failures that was not caught by my local deployment but showed in in sandbox.

Reply:
Actually I think we may just remove `datatime` field since it's not defined in the stacItem interface. Let me try to remove it and give a try.

Reply:
Just removed the `datetime` field and everything is still working. I have pushed the changed, and not sure why `netcdf-to-zarr` introduced that field.